<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Device client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>891</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="9" column="2">
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="4">
      <widget class="QPushButton" name="exit_pushButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" rowspan="2">
      <widget class="QGroupBox" name="channel1_range_groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Range:</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>20</y>
          <width>161</width>
          <height>112</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="channel1_range0_radioButton">
           <property name="text">
            <string>0.0000001 - 0.001 v</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel1_range1_radioButton">
           <property name="text">
            <string>0.001 - 1 v</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel1_range2_radioButton">
           <property name="text">
            <string>1 - 1000 v</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel1_range3_radioButton">
           <property name="text">
            <string>1000 - 1000000 v</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="channel0_checkBox">
       <property name="text">
        <string>Channel 0</string>
       </property>
      </widget>
     </item>
     <item row="17" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="channel3_status_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="2">
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="channel2_status_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="16" column="4">
      <widget class="Line" name="line_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="channel1_status_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="3">
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="4">
      <widget class="QPushButton" name="get_status_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Get status</string>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="Line" name="line_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="4">
      <widget class="QPushButton" name="stop_measure_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop measure</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="4">
      <widget class="QPushButton" name="get_result_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Get result</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2" rowspan="2">
      <widget class="QGroupBox" name="channel2_range_groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Range:</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>20</y>
          <width>161</width>
          <height>112</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="channel2_range0_radioButton">
           <property name="text">
            <string>0.0000001 - 0.001 v</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel2_range1_radioButton">
           <property name="text">
            <string>0.001 - 1 v</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel2_range2_radioButton">
           <property name="text">
            <string>1 - 1000 v</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel2_range3_radioButton">
           <property name="text">
            <string>1000 - 1000000 v</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="channel0_voltage_groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Voltage:</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>175</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="channel0_label_voltage">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="channel3_checkBox">
       <property name="text">
        <string>Channel 3</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QGroupBox" name="channel2_voltage_groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Voltage:</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>174</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="channel2_label_voltage">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="7" column="3" rowspan="2">
      <widget class="QGroupBox" name="channel3_range_groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Range:</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>20</y>
          <width>161</width>
          <height>112</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="channel3_range0_radioButton">
           <property name="text">
            <string>0.0000001 - 0.001 v</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel3_range1_radioButton">
           <property name="text">
            <string>0.001 - 1 v</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel3_range2_radioButton">
           <property name="text">
            <string>1 - 1000 v</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel3_range3_radioButton">
           <property name="text">
            <string>1000 - 1000000 v</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="7" column="0" rowspan="2">
      <widget class="QGroupBox" name="channel0_range_groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Range:</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>112</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QRadioButton" name="channel0_range0_radioButton">
           <property name="text">
            <string>0.0000001 - 0.001 v</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel0_range1_radioButton">
           <property name="text">
            <string>0.001 - 1 v</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel0_range2_radioButton">
           <property name="text">
            <string>1 - 1000 v</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="channel0_range3_radioButton">
           <property name="text">
            <string>1000 - 1000000 v</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QGroupBox" name="channel3_voltage_groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Voltage:</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>174</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="channel3_label_voltage">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QPushButton" name="connect_pushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="channel2_checkBox">
       <property name="text">
        <string>Channel 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="channel1_voltage_groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Voltage:</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>174</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="channel1_label_voltage">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="10" column="0" colspan="4">
      <widget class="QPushButton" name="start_measure_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start measure</string>
       </property>
      </widget>
     </item>
     <item row="13" column="4">
      <widget class="QPushButton" name="disconnect_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="16" column="3">
      <widget class="Line" name="line_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="channel1_checkBox">
       <property name="text">
        <string>Channel 1</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="channel0_status_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
